.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+php:
+  - 7.0
+  - 7.1
+
+services:
+  - mongodb
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - travis_retry composer self-update
+  - travis_retry composer validate
+  - pecl install -f mongodb
```

Comment:
just checking: is mongodb (not mongo) not available on Travis?

Reply:
Apparently not. If you take a look at the previous build of this PR, you'll see Composer complaining about the missing extension.